python_modules/dagster/dagster/_cli/asset.py

```diff
@@ -19,6 +20,7 @@
     recon_repository_from_origin,
 )
 from dagster._utils.interrupts import capture_interrupts
+from python_modules.dagster.dagster._cli.job import get_config_from_args
```

Comment:
Should we pull this out into a utility function to avoid importing a CLI entrypoint into another?

Reply:
I would recommend doing this (probably just putting it in dagster._cli.workspace.cli_target), as this is where other cross-command utilities live at the moment